test/Unit.purs

```diff
@@ -38,11 +42,15 @@ import Test.Ctl.Types.TokenName as Types.TokenName
 import Test.Ctl.Types.Transaction as Types.Transaction
 import Test.Ctl.UsedTxOuts as UsedTxOuts
 import Test.Ctl.Wallet.Cip30.SignData as Cip30SignData
+import Test.Spec.Runner (defaultConfig)
 
 -- Run with `spago test --main Test.Ctl.Unit`
 main :: Effect Unit
-main = launchAff_ do
-  interpret testPlan
+main = interruptOnSignal SIGINT =<< launchAff do
+  flip cancelWith (effectCanceler (exitCode 1)) do
+    interpretWithConfig
```

Comment:
There is usages of interpret in some other small modules, like MustSpendTotal. Should they be changed too?

Reply:
Or this is required only for CI-related stuff?

Reply:
No, these `main`s are not parts of our test suite (could be removed perhaps). But the MustSpendTotal suite itself is a part of unit tests